gatsby-config.js

```diff
@@ -5,9 +5,42 @@ module.exports = {
       description: 'Prizy is an app used for meetup lotteries',
       keywords: 'Prizy, Larissa Developers',
     },
+    menuLinks: [
+      {
+        name: 'Meetup',
+        link: 'https://www.meetup.com/Larissa-Developers-Meetup/',
+      },
+      {
+        name: 'Social',
+        link: '/',
+      },
+      {
+        name: 'Explore Prizy',
+        link: '/',
+      },
+    ],
   },
   pathPrefix: '/prizy_landing',
   plugins: [
+    'gatsby-plugin-sass',
+    {
+      resolve: 'gatsby-plugin-postcss',
+      options: {
+        postCssPlugins: [
+          require('postcss-preset-env')({ stage: 0 }),
+          'postcss-import',
+          'postcss-mixins',
+          'postcss-nested',
+          'postcss-simple-vars',
+        ],
+      },
+    },
+    {
+      resolve: 'gatsby-plugin-google-analytics',
+      options: {
+        trackingId: `UA-93349937-2`,
```

Comment:
Have you added Google Analytics? On which Google account have you added this property?